fastlane/lib/fastlane/server/command_executor.rb

```diff
@@ -0,0 +1,45 @@
+require 'fastlane/server/command.rb'
+
+module Fastlane
+  class CommandExecutor
+    def execute(command: nil, target_object: nil)
+      not_implemented(__method__)
+    end
+  end
+
+  class GenericCommandExecuter < CommandExecutor
```

Comment:
Comments on what the class does would be 💯, also usually we avoid having 2 classes in one file, can we add a comment on why we needed to do it?

Reply:
This class is going to be removed 🌮 updated the class that is staying with comments